flatgfa/src/packedseq.rs

```diff
@@ -1,5 +1,18 @@
+use crate::file::*;
+use crate::flatgfa;
+use crate::memfile;
+use crate::memfile::map_new_file;
+use crate::pool::*;
+use crate::FixedFamily;
+use crate::HeapFamily;
+use crate::StoreFamily;
```

Comment:
Looks like these imports (and perhaps some others) are unused. You can run `cargo build` to see a list of warnings like this.